src/client/src/content/posts/the-magic-of-tree-shaking-in-c.md

```diff
@@ -0,0 +1,57 @@
+---
+title: The Magic of Tree Shaking in C#
+category: Programming
+description: Learn how to code the classic Game of Life in JavaScript. This
```

Comment:
![image](https://github.com/ntohq/ntohq-website/assets/11604664/de875bc3-5fd7-4188-89de-6a02216ae90a)
This property is why you see duplicate descriptions on the Blog home page. I will change the description of this development blog post later.